packages/pg-gateway/src/connection.ts

```diff
@@ -117,10 +117,8 @@ export type PostgresConnectionOptions = {
 };
 
 export type MessageResponse =
-  | undefined
-  | Uint8Array
-  | Iterable<Uint8Array>
-  | AsyncIterable<Uint8Array>;
+  // biome-ignore lint/suspicious/noConfusingVoidType: this is a return type
+  void | Uint8Array | Iterable<Uint8Array> | AsyncIterable<Uint8Array>;
```

Comment:
I fixed the type here: undefined -> void.
Without this fix we were forced to `return undefined;` vs `return;`.
The biome error is only for types that aren't used as a return type, so we can safely ignore it here.